composer.json

```diff
@@ -32,5 +32,8 @@
     },
     "autoload": {
         "files": ["lib/mcrypt.php"]
+    },
+    "provide": {
+      "ext-mcrypt": "*"
```

Comment:
the value should be version, so use `0` or minimum (uppper?) php api you support

the native extension matches PHP_VERSION:

```
$ php -r '$r = new ReflectionExtension("mcrypt"); var_dump($r->getVersion());'
Command line code:1:
string(6) "7.1.25"
```

Reply:
That seemed to do the trick! I went ahead and made it provide 5.6.40 (the last 5.6 version) since mcrypt was removed in 7.0.0. I've gone ahead and made a new release as well.

Thanks!